systemtest/src/test/java/io/strimzi/systemtest/AbstractST.java

```diff
@@ -170,7 +170,7 @@ void tearDownTestCase(ExtensionContext extensionContext) throws Exception {
         // does not proceed with the next method (i.e., afterEachMustExecute()). This ensures that if such problem happen
         // it will always execute the second method.
         try {
-            assertNoCoErrorsLogged(clusterOperator.getDeploymentNamespace(), storageMap.get(extensionContext).getTestExecutionTimeInSeconds());
+            //assertNoCoErrorsLogged(clusterOperator.getDeploymentNamespace(), storageMap.get(extensionContext).getTestExecutionTimeInSeconds());
```

Comment:
Maybe you can add some comment why is it commented out? E.g. nto stable enough, needs ot be improved int he future?

Reply:
Sure, will do, thanks.